configure.scss

```diff
@@ -1,20 +1,20 @@
 $_n-ui-features: (
 	ads: true,
 	buttons: true,
-	grid: true,
-	// always in silent mode
-	colors: false,
-	// always in silent mode
-	icons: false,
-	typography: true,
+	card: false,
```

Comment:
colors & icons

Even though config here won't do anything I think this list is a good as a living list of what gets included